src/Middleware/Diagnostics/src/PublicAPI.Unshipped.txt

```diff
@@ -1,48 +1 @@
 #nullable enable
-Microsoft.AspNetCore.Builder.DeveloperExceptionPageOptions.FileProvider.get -> Microsoft.Extensions.FileProviders.IFileProvider?
```

Comment:
I think this duplication in PublicAPI files was caused by https://github.com/dotnet/roslyn-analyzers/issues/4584